src/index.html

```diff
@@ -11,7 +11,172 @@
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>HTML Form</h1>
+    <form action="https://mate-academy-form-lesson.herokuapp.com/create-application" method="post">
+      <fieldset class="form-field">
+        <legend>Personal information</legend>
+
+        <label for="surname">Surname:</label>
+        <input
+          type="text"
+          name="surname"
+          id="surname"
+          autocomplete="off"
+          class="form-input-gap"
+        >
+        <br>
+
+        <label for="name">Name:</label>
+        <input
+          type="text"
+          name="name"
+          id="name"
+          autocomplete="off"
+          class="form-input-gap"
+        >
+        <br>
+
+        <label for="age">How old are You?</label>
+        <input
+          type="number"
+          name="age"
+          id="age"
+          value="12"
+          min="1"
+          max="100"
+          class="form-input-gap"
+        >
+        <br>
+
+        <label for="dateOfBirth">Full date of birth:</label>
+        <input
+          type="date"
+          name="dateOfBirth"
+          id="dateOfBirth"
+          class="form-input-gap"
+        >
+        <br>
+
+        <label for="term">I accept the term of the agreement</label>
+        <input
+          type="checkbox"
+          name="term"
+          id="term"
+        >
+      </fieldset>
+
+      <fieldset class="form-field">
+        <legend>Registration</legend>
+
+        <label for="email">E-mail:</label>
+        <input
+          type="email"
+          name="email"
+          id="email"
+          placeholder="email@example.com"
+          required
+          class="form-input-gap"
+        >
+        <br>
+
+        <label for="password">Password:</label>
+        <input
+          type="password"
+          name="password"
+          id="password"
+          minlength="7"
+          maxlength="32"
+          required
+        >
+      </fieldset>
+
+      <fieldset class="form-field">
+        <legend>An interesting fact about you!</legend>
+
+        <div class="form-input-gap">
+          <label>
+            Do you love cats?
+            <span>
+              <input
```

Comment:
each input/select should be wrapped with a label
```suggestion
          <div>
            Do you love cats?
            <label>
              <input
```